lib/cucumber/util/colors.js

```diff
@@ -1,15 +1,15 @@
-var colors = require('colors/safe');
+function Colors (useColors) {
```

Comment:
Okay. I guess we could just put the following somewhere and set `colors.enabled` in our argument parser. Where do you think the following code should go? 
```js
colors.setTheme({
  'comment'   : 'grey',
  'failed'    : 'red',
  'passed'    : 'green',
  'pending'   : 'yellow',
  'skipped'   : 'cyan',
  'tag'       : 'cyan',
  'undefined' : 'yellow'
});
```